samples/Mvc.FileUpload/Controllers/FileController.cs

```diff
@@ -66,9 +63,8 @@ public IActionResult FormToUpload()
                 ContentDispositionHeaderValue contentDisposition;
                 ContentDispositionHeaderValue.TryParse(section.ContentDisposition, out contentDisposition);
 
-                if (MultipartRequestHelper.HasFileContentDisposition(contentDisposition))
+                if (contentDisposition.IsFileContentDisposition())
                 {
-                    var name = HeaderUtilities.RemoveQuotes(contentDisposition.Name) ?? string.Empty;
```

Comment:
Not unless reading the Name property has side effects :) `name` is not used in this `if` block